packages/gnome_shell.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gnome_shell < Package
+class Gnome_shell < Meson
   description 'Next generation desktop shell'
   homepage 'https://wiki.gnome.org/Projects/GnomeShell'
   version '41.0'
```

Comment:
Update to [44.0](https://gitlab.gnome.org/GNOME/gnome-shell/-/releases/44.0)?

Reply:
depends upon gnome_session, which now has a hard systems dep, so deprecating.